src/components/markdown_editor/markdown_editor_text_area.tsx

```diff
@@ -27,6 +28,7 @@ export type EuiMarkdownEditorTextAreaProps = TextareaHTMLAttributes<
     isInvalid?: boolean;
     fullWidth?: boolean;
     compressed?: boolean;
+    textareaClassName?: string;
```

Comment:
This just needs to be the regular `className`.
```suggestion
    className?: string;
```